tests/event_samples/lambda-url.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Where does this file come from?

Reply:
copied from [datadog-lambda-js](https://github.com/DataDog/datadog-lambda-js/blob/50a1b243aa89c973e7f7b5232f4529b312f7bc73/event_samples/lambda-function-urls.json) I wanted a test case for lambda-url and datadog-lambda-python didnt have this event sample

Reply:
Nice, great work!